### PR TITLE
runtests script: test with python2 and python3

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

**Comment by philipwinklerdelmic** [comment]
not ready to merge yet, will be tested tonight

File: util/run_intg_tests.py [hunk 1]
```diff
@@ -251,6 +252,8 @@ def main(args):
 
     parser.add_argument("--log-path", dest="logpath", default="/tmp/",
                         help="Directory where the logs will be saved")
+    parser.add_argument("--interpreter", dest="interpreter", default="/usr/bin/python",
+                        help="Python interpreter (/usr/bin/python or /usr/bin/python3)")
```
Comment (pieleric) [inline util/run_intg_tests.py]: Better be explicit by writing `/usr/bin/python2`

File: util/runtests.sh [hunk 2]
```diff
@@ -149,7 +199,8 @@ SIMPATH="$ODEMIS_DIR/install/linux/usr/share/odemis/sim/"
 
 ODMPATH="$ODEMIS_DIR/../mic-odm-yaml/" # extra microscope files
 if [ -d "$ODMPATH" ]; then
-    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" "$ODMPATH"/*/ >> "$TESTLOG" 2>&1
+    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" --interpreter "/usr/bin/python" "$ODMPATH"/*/ >> "$TESTLOG-python2" 2>&1
+	"$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" --interpreter "/usr/bin/python3" "$ODMPATH"/*/ >> "$TESTLOG-python3" 2>&1
```
Comment (pieleric) [inline util/runtests.sh]: Please use spaces for the indentation.

File: util/runtests.sh [hunk 3]
```diff
@@ -132,6 +132,56 @@ fi
...
+done
+
+if [ $failures -gt 0 ]; then
+    echo "$failures test failed. See $TESTLOG for error messages."
+else
+    echo "All tests passed"
+fi
+
+# try to clean up a bit
+sudo odemis-stop
```
Comment (pieleric) [inline util/runtests.sh]: It's a lot of duplicate code. I think it should be fine to move that code to a function (and use all the variables as globals). You could then just have one single variable PYTHON_INTERPRETER to update, or pass it as an argument.

**Comment by philipwinklerdelmic** [comment]
updated

**Comment by philipwinklerdelmic** [comment]
updated

File: util/runtests.sh [hunk 4]
```diff
@@ -149,7 +157,8 @@ SIMPATH="$ODEMIS_DIR/install/linux/usr/share/odemis/sim/"
 
 ODMPATH="$ODEMIS_DIR/../mic-odm-yaml/" # extra microscope files
 if [ -d "$ODMPATH" ]; then
-    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" "$ODMPATH"/*/ >> "$TESTLOG" 2>&1
+    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" --interpreter "/usr/bin/python" "$ODMPATH"/*/ >> "$TESTLOG-python2" 2>&1
```
Comment (pieleric) [inline util/runtests.sh]: Let's write explicitly python2, to be 100% sure the right Python starts.

Comment (philipwinklerdelmic) [reply util/runtests.sh]: done

**Pull request merged**